### DISCUSSION TITLE
How to install

### DISCUSSION THREAD

**Discussion by ellukitas-123** [micjahn/ZXing.Net#317] [Q&A]
Hi,

I'm trying to import this package to my unity project but i can't

Someone can tell me how to import to my unity project please?

**Comment by micjahn** [comment]
There is an old unity demo inside the repository:
https://github.com/micjahn/ZXing.Net/tree/master/Clients/UnityDemo
Perhaps it helps you.

**Comment by MoutonRedstone** [comment]
I have the same question, because the unity file is hidden somewhere

**Reply by micjahn** [reply]
What do you mean by "is hidden"? The zxing assembly for unity is included in the zip archive which you can get from the download section here.
